etc/kayobe/ipa.yml

```diff
@@ -5,7 +5,7 @@
 # Ironic Python Agent (IPA) image build configuration.
 
 # Whether to build IPA images from source.
-#ipa_build_images:
+ipa_build_images: True
```

Comment:
This does change the workflow a bit - now you need to build an image, both for bifrost & overcloud ironic. I'd at least avoid putting this in wallaby.

My medium term goal is to build IPA images and put them on Ark, then just include config to pull them down in here. The build flag would go in the `ci-builder` environment.

Reply:
I removed `ipa_build_images` for now, it is up to the operator to enable it.